static/js/subs.js

```diff
@@ -1117,8 +1117,23 @@ $(function () {
         sub_arrow.removeClass('icon-vector-chevron-up');
         sub_arrow.addClass('icon-vector-chevron-down');
     });
+
+    //Set the width of subscriptions status box with respect to size of
+    //middle column.
+    function set_subscription_status_width() {
+        var column_middle_width = $('.column-middle').width();
+        $('#subscriptions-status').width(0.8*column_middle_width);
+    }
+
+    $(window).resize(function () {
+        set_subscription_status_width();
+    });
+
+    set_subscription_status_width();
```

Comment:
Would it work to do this in pure CSS by making the templates/zerver/subscriptions.html  `<div class="subscriptions">` container div have the `.column-middle` class (or something similar)?  This still seems like it's more complicated than the "an element inside the larger middle column element" should need to be...